python/res/enkf/queue_config.py

```diff
@@ -57,12 +58,25 @@ def max_submit(self):
 
     @property
     def queue_name(self):
-        return self._queue_name()
+        return self.driver.get_option(ConfigKeys.LSF_QUEUE_NAME_KEY)
+
+    @property
+    def queue_system(self):
+        """The queue system in use, e.g. LSF or LOCAL"""
+        return self._queue_system()
 
     @property
     def driver(self):
-        return self._queue_driver(self.queue_name).setParent(self)
+        return self._queue_driver(self.queue_system).setParent(self)
 
     @property
     def job_script(self):
         return self._get_job_script()
+
+    @property
+    def lsf_resource(self):
```

Comment:
This should in my opinion explicitly ask for the LSF driver, and then return resource. The following situation is realistic:

1. The user has configured to use the `LOCAL` system.
2. When asking for the `lsf_resource` the queue config will look up the active - i.e. "LOCAL" system, and then query that for lsf resource -> Exception.

Reply:
done